src/unread/UnreadCards.js

```diff
@@ -34,14 +34,12 @@ class UnreadCards extends PureComponent<Props> {
   };
 
   render() {
-    const { conversations, unreadStreamsAndTopics, ...restProps } = this.props;
-    type Card =
-      | UnreadStreamItem
-      | { key: 'private', data: Array<$PropertyType<PmConversationList, 'props'>> };
+    const { conversations, unreadStreamsAndTopics } = this.props;
+    type Card = UnreadStreamItem | { key: 'private', data: Array<PmConversationData[]> };
     const unreadCards: Array<Card> = [
       {
         key: 'private',
-        data: [{ conversations, ...restProps }],
+        data: [conversations],
```

Comment:
Hmm, can this really work with how this data gets used below?
```
          section.key === 'private' ? (
            <PmConversationList {...item} />
```

How does it know to call the prop `conversations`?

Reply:
This can't possibly have worked. I think when I tested it last night I wasn't actually running the right version of the code. :sweat: